integration-tests/sync-tests/sync-replica.spec.ts

```diff
@@ -0,0 +1,124 @@
+import chai, { expect } from 'chai'
+import { Wallet, Contract, ContractFactory, providers } from 'ethers'
+import { ethers } from 'hardhat'
+import { injectL2Context } from '@eth-optimism/core-utils'
+
+import { sleep, l2Provider, replicaProvider } from '../test/shared/utils'
+import { OptimismEnv } from '../test/shared/env'
+import { DockerComposeNetwork } from '../test/shared/docker-compose'
+
+describe('Syncing a replica', () => {
+  let env: OptimismEnv
+  let wallet: Wallet
+  let replica: DockerComposeNetwork
+  let provider: providers.JsonRpcProvider
+
+  const sequencerProvider = injectL2Context(l2Provider)
+
+  /* Helper functions */
+
+  const startReplica = async () => {
+    // Bring up new replica
+    replica = new DockerComposeNetwork(['l2_dtl', 'replica'])
+    await replica.up({
+      commandOptions: ['--scale', 'l2_dtl=1', '--scale', 'replica=1'],
+    })
+
+    // Wait for replica to be looping
+    let logs = await replica.logs()
+    while (!logs.out.includes('Starting Sequencer Loop')) {
+      console.log(logs)
+      await sleep(500)
+      logs = await replica.logs()
+    }
```

Comment:
Why does a Replica run the sequencer loop? Shouldn't it be running the verifier loop?

Reply:
hmm i'm not sure but this does work 🤔 will investigate, @tynes is there a better log to grep for here?

Reply:
You should be able to poll the RPC to do it in a generalized way. Note that if this works, then its not running as a verifier or you are getting logs from the wrong container because the log does say `Starting Verifier Loop`

https://github.com/ethereum-optimism/optimism/blob/86708bb5758cd2b647b3ca2be698beb5aa3af81f/l2geth/rollup/sync_service.go#L319

Reply:
@tynes corrected the config and updated the log i grep for, but lmk if you think there's a better way to query the replica to ensure it's up!

Reply:
I feel like polling the RPC would prevent an unknowning future person from removing/altering that logline in a way that it causes these tests to fail